.mailmap

```diff
@@ -1397,6 +1398,7 @@ dustyrockpyle <dustyrockpyle@gmail.com>
 elvis-sik <e.sikora@grad.ufsc.br>
 erdOne <36414270+erdOne@users.noreply.github.com>
 faizan2700 <syedfaizan824@gmail.com>
+fauzank339 <50906187+fauzank339@users.noreply.github.com>
```

Comment:
Only one name should be added to .mailmap. Which name do you want to be in the AUTHORS file?